Makefile

```diff
@@ -161,9 +161,9 @@ misspell-correction:
 
 .PHONY: tidy
 tidy:
-	go mod tidy -compat=1.17
-	cd tests && go mod tidy -compat=1.17
-	cd internal/tools && go mod tidy -compat=1.17
```

Comment:
Should we just bump to `-compat=1.18` like in contrib?

Reply:
Sorry I just barely missed this before merging. 

Our collector project is at 1.19, I thought we should go with what our collector go.mod version has. 

Thoughts?

Reply:
from the docs the default behavior is to use 1 version below the module version so they're the same behavior as stripping I think.  Curious why the compat needed to be removed?

Reply:
That is a useful bit of information to know. The -compat=1.17 was only out of date so I removed it, now I'll set -compat=1.18 in my next PR.